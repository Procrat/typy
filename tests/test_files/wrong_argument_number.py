hex(5, 6)
