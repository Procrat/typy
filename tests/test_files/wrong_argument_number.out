hex expected 1 parameters, but received 2
